sentences/en/climate_HassClimateGetTemperature.yaml

```diff
@@ -2,18 +2,19 @@ language: "en"
 intents:
   HassClimateGetTemperature:
     data:
-      # What is...
+      # Get temperature of a climate in the same area as a satellite device
+      - sentences:
+          - "<what_is> [the] [current] <temp> [[in] here]"
+          - "how (hot|cold|warm|cool) is it [[in] here]"
+        requires_context:
+          area:
+            slot: true
+
+      # Get temperature of a climate in an area or with a name
       - sentences:
           - "<what_is> [the] [current] <temp> [of] <name>"
-          - "<what_is> [the] [current] <temp> [in <area>]"
+          - "<what_is> [the] [current] <temp> in <area>"
```

Comment:
Instead of `in <area>`, I think `<in_area_floor>` will be more expansive.

Reply:
HassClimateGetTemperature doesn't support floor at the moment from what I can tell (https://github.com/home-assistant/core/blob/f8399b2c0fa6e71fbfc2ff28d96bf8f7bf133364/homeassistant/components/climate/intent.py#L18) , so I've kept this as just area.

Reply:
Would be good, but I’m ok merging the PR supporting what’s in Core today.

Reply:
Could still use `<in>` rather than the literal to cover other relevant prepositions. That way it covers “what is the temperature of the living room” as well.